controllers/machine_ab_pruning_actor.py

```diff
@@ -47,5 +47,12 @@ def __find_best_solution_rec(self, depth, game, alpha=-100):
                 alpha = sol
         return alpha, alphapawn, alphamove
 
-    def __get_heuristic(self, game):
+    def _get_heuristic(self, game):
         raise NotImplementedError("Abstract method.")
+
+    def _get_fields_delta(self, board, current_player):

```

Comment:
rename to _get_pawn_delta

Reply:
correct naming but hat to adjust comment. its not only the count of pawns.